### PR TITLE
chore: remove all karma related setting, update npm modules

### PR DESCRIPTION
**Pull request by aweiss-dev**
…build for ansi-regex

## Pull Request Checklist

- [ ] My code is covered by tests
- [ ] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: jest.config.base.js [hunk 1]
```diff
@@ -20,7 +20,11 @@ module.exports = {
     clearMocks: true,
     testMatch: ["**/?(*.)+(spec|test).+(ts|tsx)", "test/**/*.+(ts|tsx)",],
     transform: {
-        "^.+\\.(ts|tsx)$": "@swc/jest"
+        "^.+\\.(ts|tsx)$": "@swc/jest",
+        "^.+\\.(js|jsx)$": "babel-jest"
     },
+    transformIgnorePatterns: [
+        "!node_modules/"
```
Comment (atomrc) [inline jest.config.base.js:27]: I don't understand this, you want to `ignore` whatever is `not node_modules`. Which means that you ignore the sources and transpile only what is in `node_modules`? 🙃

File: packages/commons/package.json [hunk 2]
```diff
@@ -1,22 +1,23 @@
 {
   "dependencies": {
-    "ansi-regex": "5.0.1",
+    "@babel/preset-env": "^7.19.4",
+    "ansi-regex": "6.0.1",
```
Comment (atomrc) [inline packages/commons/package.json:4]: If this causes problem, let's upgrade it in a different PR to avoid mixing topics (and remove the `babel` config for now)

Comment (aweiss-dev) [reply packages/commons/package.json]: its gone. removed babel everywhere besides react-ui-kit where it is actively in use for building

File: packages/commons/package.json [hunk 3]
```diff
@@ -1,22 +1,23 @@
 {
   "dependencies": {
-    "ansi-regex": "5.0.1",
+    "@babel/preset-env": "^7.19.4",
```
[the remark this answers is not in the document]
Comment (aweiss-dev) [reply packages/commons/package.json]: its gone :)

File: jest.config.base.js (hunk 1 above)
Comment (aweiss-dev) [reply jest.config.base.js]: Fixed

File: jest.config.base.js [hunk 4]
```diff
@@ -20,7 +20,10 @@ module.exports = {
     clearMocks: true,
     testMatch: ["**/?(*.)+(spec|test).+(ts|tsx)", "test/**/*.+(ts|tsx)",],
     transform: {
-        "^.+\\.(ts|tsx)$": "@swc/jest"
+        "^.+\\.(ts|tsx|jsx)$": "@swc/jest",
     },
+    transformIgnorePatterns: [
+        "node_modules/"
+    ],
```
Comment (atomrc) [inline jest.config.base.js:27]: Normally this should be the default config, no need to be explicit about it. Can you check?

File: packages/commons/babel.config.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    "presets": [
+        "@babel/preset-env"
+    ]
+};
```
Comment (atomrc) [inline packages/commons/babel.config.js:5]: This can probably go now, right?

File: packages/core/package.json [hunk 6]
```diff
@@ -16,19 +16,13 @@
     "http-status-codes": "2.2.0",
     "idb": "7.1.0",
     "logdown": "3.3.1",
-    "long": "4.0.0",
```
Comment (atomrc) [inline packages/core/package.json:19]: I do believe we use `long` in the core, why did you remove it? 🤔

**Review by atomrc — APPROVED** [review]
Looks good. Can you still check the few questions I asked?
Good job man, thanks 😎

File: packages/commons/babel.config.js (hunk 5 above)
Comment (aweiss-dev) [reply packages/commons/babel.config.js]: gone

File: packages/core/package.json (hunk 6 above)
Comment (aweiss-dev) [reply packages/core/package.json]: i added it back in
